src/main/java/org/asamk/signal/json/JsonMention.java

```diff
@@ -0,0 +1,18 @@
+package org.asamk.signal.json;
+
+import java.util.UUID;
+import org.whispersystems.signalservice.api.messages.SignalServiceDataMessage;
+
+public class JsonMention {
+
+    UUID uuid;
+    int start;
+    int length;
+
+    JsonMention(SignalServiceDataMessage.Mention mention) {
+        this.uuid = mention.getUuid();
```

Comment:
That's a good idea. I noticed in multiple places (Mentions, quote's authors, some members from `listGroups` output, etc) it shows a UUID instead of a name. For Reactions we did this with by resolving the `SignalServiceAddress` with the manager and other things, but Mentions only [provides us](https://github.com/signalapp/Signal-Android/blob/9e7c55847e196165940ccb93c7bf55adbd3dac4a/libsignal/service/src/main/java/org/whispersystems/signalservice/api/messages/SignalServiceDataMessage.java#L558) the UUID.

I nearly gave up on adding this, but I finally found something that gets the mentioned person's phone number from the UUID. Do you want me to have both the UUID and phone number in the output or just the number? I'll also modify the plain-text outputs to show the phone number too.